test/models/backend_api_test.rb

```diff
@@ -21,7 +21,6 @@ def test_default_api_backend
     @account.stubs(:provider_can_use?).with(:apicast_v1).returns(true)
     @account.stubs(:provider_can_use?).with(:apicast_v2).returns(true)
     @account.expects(:provider_can_use?).with(:proxy_private_base_path).at_least_once.returns(false)
-    @account.expects(:provider_can_use?).with(:api_as_product).at_least_once.returns(false)
```

Comment:
I see that this used to be the scenario of non-APIAP, my question here is why do we keep this test? do we want APIAP (our current scenario) to behave the exact same way? if so, is this test repeated then?

Reply:
I see.. well the tests is for BackendAPI so it should only apply to an APIAP scenario, right? I think this line doesn't do anything since the test passes anyway.